### ISSUE TITLE
Better highlighting for nav bar

### ISSUE THREAD

**Issue by aaronsn** [SatcherInstitute/het-frontend#35]
- More clearly differentiate the button highlight color from the app bar color
- Show the currently viewed page in a different color to indicate where the user currently is

**Comment by aaronsn** [comment]
@kkatzen

**Comment by kkatzen** [comment]
I was assuming these sorts of details would be addressed with Ken's mocks. I'm not sure it's worth addressing any particular UI concerns until we have more of an idea of what we're heading overall UI-wise.

**Comment by aaronsn** [comment]
Yeah, no need to do this right away (hence why I tagged it as low priority), mostly just filing this so we don't forget. That being said, showing the currently viewed page more clearly is a standard UI polish thing that almost all sites do - I think this is pretty uncontroversial to do in some way regardless of how the mocks turn out
